dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/transport/CodecSupport.java

```diff
@@ -36,6 +36,7 @@
     private static final Logger logger = LoggerFactory.getLogger(CodecSupport.class);
     private static Map<Byte, Serialization> ID_SERIALIZATION_MAP = new HashMap<Byte, Serialization>();
     private static Map<Byte, String> ID_SERIALIZATIONNAME_MAP = new HashMap<Byte, String>();
+    private static Map<String, Byte> Name_ID_MAP = new HashMap<String, Byte>();
```

Comment:
**SERIALIZATIONNAME_ID_MAP** is better, in my opinion.

Reply:
good advice.